src/js/utils/download.js

```diff
@@ -0,0 +1,15 @@
+export function downloadSvgoConfig(filename, text) {
+  const element = document.createElement('a');
```

Comment:
I'm wondering, isn't there a cleaner way without adding the extra `a` tag?

Reply:
Rewrote the code so that the link to the configuration was always on the page and was updated when the configuration was changed